pkg/xds/monitor.go

```diff
@@ -7,12 +7,9 @@ import (
 
 	metricsservice "github.com/envoyproxy/go-control-plane/envoy/service/metrics/v3"
 	prom "github.com/prometheus/client_model/go"
-)
 
-type Scaler interface {
-	ScaleToZero(cluster string)
-	ScaleFromZero(cluster string)
-}
```

Comment:
It is better keep the `Scaler` interface here, because it is defined and required by `MonitorServer` and it is not limited to be k8s implementation only